app/main.py

```diff
@@ -1,3 +1,81 @@
+from typing import Union
```

Comment:
The import statement is correct, but ensure that the `Union` type is necessary for all methods where it's used. Consider if `float` alone would suffice in some cases.